dags.yaml

```diff
@@ -1280,3 +1280,23 @@ bqetl_review_checker:
   tags:
     - impact/tier_1
     - repo/bigquery-etl
+
+bqetl_mobile_feature_usage:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - rzhao@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: rzhao@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2023-10-24'
+  description: schedule run for mobile feature usage tables
+  repo: bigquery-etl
+  schedule_interval: 0 4 * * *
+  tags:
+    - impact/tier_3
+    - repo/bigquery-etl
```

Comment:
```suggestion
```

Reply:
This tag will be automatically added as part of the generation in this specific repo.